multi-node/libvirt/kube-master/openstack/latest/user_data

```diff
@@ -0,0 +1,616 @@
+#!/bin/bash
```

Comment:
Does anything differ about this file than multi-node/generic? (same for the kube-worker)